draft-barnes-sframe-mls.md

```diff
@@ -64,8 +64,42 @@ properties apply to real-time media as well.  In the remainder of this document,
 we define how to use the secrets produced by MLS to generate the keys required
 by SFrame.
 
-[[ OPEN ISSUE: We could define an MLS extension that would provide negotiation
-of SFrame parameters, notably the ciphersuite and the value E defined below. ]]
+# SFrame Parameter Negotiation
+
+In order to interoperate, the sender and receiver(s) of an SFrame payload need
+to agree on two parameters:
+
+* The SFrame ciphersuite
+* The number of bits `E` used to signal the epoch
+
+These parameters can be negotiated in MLS using the `sframe_parameters`
+extension.  An MLS participant advertises its supported ciphersuites in its
+KeyPackage.  The creator of the group chooses the values of these parameters for
+the group (possibly based on a set of KeyPackages) and advises them to new
+joiners in Welcome messages.
+
+```
+uint16 SFrameCipherSuite;
+
+struct {
+  SFrameCipherSuite cipher_suites<0..255>;
+} SFrameCapabilities;
+
+struct {
+  SFrameCipherSuite cipher_suite;
```

Comment:
should this be SFrameCapabilities instead ?

Reply:
No.  SFrameCapabilities is a **list** of supported ciphersuites; SFrameParameters is a **single** suite that will be used for the group.